guides/frameworks/tauri.mdx

```diff
@@ -111,7 +113,7 @@ async fn get_all_items() -> Result<Vec<Item>> {
     let conn = db.connect()?;
 
     let mut results = conn
-        .query("SELECT * FROM items", ())
+        .query("SELECT * FROM table_name", ())
```

Comment:
@xinnks I'm open to using a specific table name if you think it's makes more sense... or better... we use a real table name and include the example schema or SQL dump for people to create a db from if following... 

Maybe we do that in an update though. We don't need that for this initial release.